### PR TITLE
Reuse t/test_postgresql in container test as well

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
hm, in this case I guess we can trust the codecov report. The tests do not fail but are mostly a noop because the database is not actually picked up, e.g. in https://travis-ci.org/os-autoinst/openQA/jobs/556751840#L865 . Let's see if I need to pass TEST_PG properly or something.

**Comment by kraih** [comment]
Wow, that is not good. But finally @codecov has proven itself useful!

File: docker/openqa/entrypoint.sh [hunk 1]
```diff
@@ -16,42 +16,21 @@ export LC_ALL="en_US.UTF-8"
...
     MOJO_TMPDIR=$(mktemp -d)
     export MOJO_TMPDIR
     export OPENQA_LOGFILE=/tmp/openqa-debug.log
-
     ([ "$INSTALL_FROM_CPAN" -eq 1 ] && eval "$(perl -I ~/perl5/lib/perl5/ -Mlocal::lib)") || true
 }
 
 run_as_normal_user
+export USER="${NORMAL_USER}"
+t/test_postgresql
```
Comment (andrii-suse) [inline docker/openqa/entrypoint.sh]: this command will setup DB but will not export TEST_PG; e.g. command like below will do the job 
```$(t/test_postgresql | grep TEST_PG=)```

Comment (okurz) [reply docker/openqa/entrypoint.sh]: true, thats to be fixed. thx. Updated.

**Comment by andrii-suse** [comment]
It looks it did export almost properly, just string is double quoted now ` DBI Connection failed: Can't connect to data source ''DBI:Pg:dbname=openqa_test;host=/tmp/tmp.eo8D1rDpox''` <- notice double single quotes, which I guess is the problem

File: docker/openqa/entrypoint.sh (hunk 1 above)
Comment (andrii-suse) [reply docker/openqa/entrypoint.sh]: I've tested and this seems working ```eval "$(t/test_postgresql | grep TEST_PG=)"```

**Pull request merged**